content/support-training.html

```diff
@@ -69,6 +69,13 @@ <h2>Commercial support</h2>
               <a href="https://acend.ch/en/trainings/prometheus/" class="btn btn-training btn-default btn-large" target="_blank">
                 <img src="../assets/commercial-support-logos/acend.svg" alt="acend">
               </a>
+
+            <div class="col-md-3 col-sm-6 doc-content tc-col">
```

Comment:
This section is in the wrong position, it is currently nested within the Acend logo section, it needs to go after the `</div>` of that one. Also, could you make sure that the indentation is just like for the other logo sections?

Reply:
Completed

Reply:
Thanks, the opening div line is still indented differently than the closing though?